test/test/java/DemoQATest/DemoQaTest.java

```diff
@@ -0,0 +1,144 @@
+package DemoQATest;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.*;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class DemoQaTest {
+
+    public static final String DOUBLE_CLICK_MESSAGE = "doubleClickMessage";
+    public static final String SHOW_SMALL_MODAL = "showSmallModal";
+    private static final By ELEMENTS_CARD = By.xpath("//div[@class='card-body']/h5[text()='Elements']");
+    private static final By BUTTONS_MENU_ITEM = By.id("item-4");
```

Comment:
stick to one style, or you put `String` used in `By` locators and move it to constants. or you move `By`to constatnts